src/mono/sample/wasm/browser2/declarations/interop.d.ts

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// This file defines various types related with the interaction with Dotnet interop
+
+// TODO find better types than the anys, however it will do for now
+
+export interface BINDING { // Should BINDING be renamed?
```

Comment:
Yes, and it should belong to DotNet namespace.
Also all the methods here should be renamed to camelCase as usual in JS.

Also I would like to understand if this is public contract or just internal API for Mono team.

Reply:
Ok. I also didn't yet add the library_mono.js exported functions nor those from dotnet_support.js. Perhaps I can ask this during the team meeting today.